Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1864,13 +1864,27 @@ void MediaPlayerPrivateGStreamer::configureMediaStreamAudioTracks()
 #endif
 }
 
-void MediaPlayerPrivateGStreamer::setVolume(float volume)
+void MediaPlayerPrivateGStreamer::setVolumeLocked(bool volumeLocked)
 {
+    if (m_volumeLocked == volumeLocked)
+        return;
+
+    GST_DEBUG_OBJECT(pipeline(), "Volume is now %s", m_volumeLocked ? "locked" : "unlocked");
+    m_volumeLocked = volumeLocked;
```

Comment:
I guess you want to swap this two lines.

Reply:
indeed 😂